.travis.yml

```diff
@@ -106,6 +106,9 @@ matrix:
     - python: 3.4
       env:
         - TEST_ASCII="true"
+    - python: 3.4
+      env:
+        - TEST_PYCOSAT="true"
```

Comment:
Is there any way we can avoid adding yet another entry to the Travis test matrix?